.env.sepolia

```diff
@@ -7,3 +7,4 @@ MINTER_ADDRESS='0x714c7985B073b1177356560631A30D24F60f9241'
 TRANSACTION_URL=https://sepolia.etherscan.io/tx/{{hash}}
 TOKEN_SYMBOL=BOB
 SHIELDED_TOKEN_SYMBOL=shBOB
+DELEGATED_PROVER_URL=''
```

Comment:
Do we have any worked delegate prover for the test purposes?

Reply:
Yes, of course. https://github.com/zkBob/zkbob-prover

Reply:
If you mean running service then no, we don't have. I tested with local service.